lib/pdf_fill/forms/va686c674.rb

```diff
@@ -1870,6 +1873,54 @@ def expand_veteran_ssn
           @form_data['veteran_ssn']['ssn' + (i + 1).to_s] = veteran_ssn
         end
       end
+
+      def merge_addendum_helpers
+        addendum_text = add_household_income
+
+        # income question when adding spouse
+        spouse_name = combine_full_name(@form_data.dig('dependents_application', 'spouse_information', 'full_name'))
+        spouse_income = @form_data.dig('dependents_application', 'does_live_with_spouse', 'spouse_income')
+        addendum_text += add_dependent_income(spouse_name, spouse_income)
+
+        # income questions when adding children
+        children_to_add = @form_data.dig('dependents_application', 'children_to_add')
+        addendum_text += add_dependents(children_to_add, 'child_income')
+
+        # income question when reporting a divorce
+        spouse_name = combine_full_name(@form_data.dig('dependents_application', 'report_divorce', 'full_name'))
+        spouse_income = @form_data.dig('dependents_application', 'report_divorce', 'spouse_income')
+        addendum_text += add_dependent_income(spouse_name, spouse_income)
+
+        # income questions when reporting deaths
+        deaths = @form_data.dig('dependents_application', 'deaths')
+        addendum_text += add_dependents(deaths, 'dependent_income')
+
+        @form_data['addendum'] = addendum_text
+      end
+
+      def add_household_income
+        net_worth = @form_data.dig('dependents_application', 'household_income')
+        'Did the household have a net worth greater than $130,773 in the last tax year? ' + format_boolean(net_worth)
+      end
+
+      def add_dependent_income(dependent_name, dependent_income)
+        return '' if dependent_name.blank?
+
+        "\n\nDid #{dependent_name} have an income in the last 365 days? " + format_boolean(dependent_income)
```

Comment:
Can you use string interpolation here instead of + format_boolean(dependent_income) <-- same for the other lines too

Reply:
@LindseySaari good catch!  I refactored a couple times and miss that part - thank you!